core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -915,7 +915,7 @@ public static boolean defaultUseHttp2Preface() {
      * {@link ClientFactoryBuilder#preferHttp1(boolean)}.
      *
      * <p>This flag is disabled by default. Specify the
-     * {@code -Dcom.linecorp.armeria.defaultPreferHttp1=true} JVM option to enable it.
+     * {@code -Dcom.linecorp.armeria.preferHttp1=true} JVM option to enable it.
```

Comment:
`defaultPreferHttp1` seems correct. Would you fix `SystemPropertyFlagsProvider` to handle both `preferHttp1` and `defaultPreferHttp1` to avoid breaking changes? 
https://github.com/line/armeria/blob/9535f8ddf1ae6a0fe0c03ed66a3ff38db01f6827/core/src/main/java/com/linecorp/armeria/common/SystemPropertyFlagsProvider.java#L278